.gitignore

```diff
@@ -5,3 +5,4 @@ dist/
 .env
 .eslintcache
 tsconfig.tsbuildinfo
+ coverage/
```

Comment:
Thank you for adding this to `.gitignore`. 👍 However, all the content in that directory was still added to this branch/PR, so will need to be removed.

Reply:
@aiwilliams  I have removed all the files related to the code coverage